app/models/institutions_archive.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require_dependency 'institution'
+
+class InstitutionsArchive < Institution
+  self.table_name = 'institutions_archives'
+
+  # class methods
+  # rubocop:disable Metrics/MethodLength
+  def self.archive_previous_versions(user)
+    production_version = Version.current_production.number
+    previous_version = Version.previous_production.number
+
+    begin
+      ActiveRecord::Base.transaction do
+        create_archives(previous_version, production_version)
+        Institution.where('version >= ? and version < ?', previous_version, production_version).delete_all
+      end
+    rescue ActiveRecord::StatementInvalid => exception
+      notice = 'There was an error occurring at the database level'
+      error_msg = exception.original_exception.result.error_message
+      Raven.tags_context(
+        user: user,
+        date_time: Time.current,
```

Comment:
I don't think we need these two tags above:
* Not sure what useful data `user` gets us that would help us troubleshoot an error.  Plus we should be specifying primitive values, not Ruby objects
* All the Sentry exceptions are already timestamped.

Reply:
removed